src/info-label.scss

```diff
@@ -0,0 +1,128 @@
+@import "./settings";
+@import "./mixins";
+@import "./functions";
+
+/*!
+.fd-badge+(( --success | --warning | --error ), (--pill | --filled))
+*/
+$block: #{$fd-namespace}-info-label;
+
+.#{$block} {
+  /*!
+  the below hardcoded colour values will be changed once we recieve variable creation from the fundamental team
+  */
+  $sapAccend1: #fdf3e6;
```

Comment:
Use kebab-case

Reply:
Let's not hardcode the values. The new variables will be released next week

Reply:
this will be addressed after the new release.